### PR TITLE
[stable] core.internal.dassert: Fix miniFormat for structs with non-copyable fields

### PR DESCRIPTION
**Pull request by kinke**
This is blocking dlang/dmd#11005.

### CODE REVIEW

File: src/core/internal/dassert.d [hunk 1]
```diff
@@ -156,10 +156,10 @@ private string miniFormat(V)(const ref V v)
         string msg = "[";
         foreach (k, ref val; v)
         {
-            if (i++ > 0)
```
Comment (MoonlightSentinel) [inline src/core/internal/dassert.d:159]: Unrelated change?

Comment (kinke) [reply src/core/internal/dassert.d:159]: Yeah, noticed this inconsistency (would have only shown first 29 items) and fixed it.

**Comment by kinke** [comment]
> Could use a test

The whole module could, but I didn't want to start adding the first unittest either. ;) - It's tested by the DMD PR.
